diff --git a/lib/asn1/typedefs.h b/lib/asn1/typedefs.h
index 756a629..0ff5278 100644
--- a/lib/asn1/typedefs.h
+++ b/lib/asn1/typedefs.h
@@ -32,6 +32,7 @@ typedef unsigned char u_char;
 #else
 
 #include <stdio.h>
+#include <stdbool.h>
 #include <stdlib.h>
 #include <stdarg.h>
 #include <sys/stat.h>
@@ -52,7 +53,6 @@ typedef unsigned char u_char;
 
 #endif
 
-typedef unsigned char bool;
 typedef unsigned int u_int;
 
 #define DBG1(s...)
